example/src/components/PaymentScreen.tsx

```diff
@@ -26,18 +27,21 @@ const PaymentScreen: React.FC<Props> = ({ paymentMethod, children }) => {
     initialize();
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
+
   return loading ? (
     <ActivityIndicator size="large" style={StyleSheet.absoluteFill} />
   ) : (
-    <ScrollView
-      accessibilityLabel="payment-screen"
-      keyboardShouldPersistTaps="always"
-      style={styles.container}
-    >
-      {children}
-      {/* eslint-disable-next-line react-native/no-inline-styles */}
-      <Text style={{ opacity: 0 }}>appium fix</Text>
-    </ScrollView>
+    <StripeContainer keyboardShouldPersistTaps={false}>
```

Comment:
`false, deprecated, use 'never' instead` - https://reactnative.dev/docs/scrollview#keyboardshouldpersisttaps

Reply:
it's a custom property for StripeContainer, just the same prop name as for ScrollView to be more consistent but we don't need to handle the same cases as scrollView does, thats why I set boolean instead of `always | never | handled`

Reply:
Okay, sounds good, thanks!